api/src/main/java/ai/djl/training/EasyTrain.java

```diff
@@ -29,6 +29,8 @@
 /** Helper for easy training of a whole model, a trainining batch, or a validation batch. */
 public final class EasyTrain {
 
+    public static final EasyTrainer TRAINER = new EasyTrainer();
```

Comment:
In the description, it says "non-final class EasyTrainer", but here it is final. Is this an error?
I'm wondering how this change enables users to modify EasyTrain. Is there a brief demo?

Reply:
The class `EasyTrainer` is not final, but this instance `TRAINER` which serves as the default `EasyTrainer` is final.